api.raml

```diff
@@ -320,8 +330,7 @@ schemas:
       responses:
         200:
           description:
```

Comment:
No, mine didn't either. I have hit errors (I don't remember whether it was in my IDE or elsewhere) for things similar to this, though... that's all.